pytest_tests/testsuites/services/http_gate/test_http_system_header.py

```diff
@@ -0,0 +1,413 @@
+    @pytest.mark.parametrize(
+        "object_size",
+        [pytest.lazy_fixture("simple_object_size")],
+        ids=["simple object"],
+    )
+    def test_unable_put_expired_epoch(self, create_container: str, object_size: int):
+        headers = attr_into_str_header_curl(
+            {"Neofs-Expiration-Epoch": str(self.return_curr_epoch() - 1)}
+        )
+        file_path = generate_file(object_size)
+        with allure.step(
+            "Put object using HTTP with attribute Expiration-Epoch where epoch is expired"
+        ):
+            upload_via_http_gate_curl(
+                cid=create_container,
+                filepath=file_path,
+                endpoint=self.cluster.default_http_gate_endpoint,
+                headers=headers,
+                error_pattern="object has expired",
+            )
+
+    @allure.title("[negative] attempt to put object with negative Neofs-Expiration-Duration")
+    @pytest.mark.parametrize(
+        "object_size",
+        [pytest.lazy_fixture("simple_object_size")],
+        ids=["simple object"],
+    )
+    def test_unable_put_negative_duration(self, create_container: str, object_size: int):
+        headers = attr_into_str_header_curl({"Neofs-Expiration-Duration": "-1h"})
+        file_path = generate_file(object_size)
+        with allure.step(
+            "Put object using HTTP with attribute Neofs-Expiration-Duration where duration is negative"
+        ):
+            upload_via_http_gate_curl(
+                cid=create_container,
+                filepath=file_path,
+                endpoint=self.cluster.default_http_gate_endpoint,
+                headers=headers,
+                error_pattern=f"{EXPIRATION_DURATION_HEADER} must be positive",
+            )
+
+    @allure.title(
+        "[negative] attempt to put object with Neofs-Expiration-Timestamp value in the past"
+    )
+    @pytest.mark.parametrize(
+        "object_size",
+        [pytest.lazy_fixture("simple_object_size")],
+        ids=["simple object"],
+    )
+    def test_unable_put_expired_timestamp(self, create_container: str, object_size: int):
+        headers = attr_into_str_header_curl({"Neofs-Expiration-Timestamp": "1635075727"})
+        file_path = generate_file(object_size)
+        with allure.step(
+            "Put object using HTTP with attribute Neofs-Expiration-Timestamp where duration is in the past"
+        ):
+            upload_via_http_gate_curl(
+                cid=create_container,
+                filepath=file_path,
+                endpoint=self.cluster.default_http_gate_endpoint,
+                headers=headers,
+                error_pattern=f"{EXPIRATION_TIMESTAMP_HEADER} must be in the future",
+            )
+
+    @allure.title(
+        "[negative] Put object using HTTP with attribute Neofs-Expiration-RFC3339 where duration is in the past"
+    )
+    @pytest.mark.parametrize(
+        "object_size",
+        [pytest.lazy_fixture("simple_object_size")],
```

Comment:
if this is the only one parameter, why not just use it as arg of the test

```
@pytest.mark.parametrize(
        "object_size",
        [pytest.lazy_fixture("simple_object_size")],
        ids=["simple object"],
    )
    def test_unable_put_expired_rfc(self, create_container: str, object_size: int):
```
just 

```
def test_unable_put_expired_rfc(self, create_container: str, simple_object_size: int):
```

Reply:
done